packages/halo/src/react/__specs__/index.spec.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
```

Comment:
The commit msg is for theme, so I'm not sure if conventional commit will bump the version on the halo package or not.  But it probably doesn't matter anyway, given that this is a non-functional refactor.